src/ts/components/seekbarlabel.ts

```diff
@@ -110,6 +105,12 @@ export class SeekBarLabel extends Container<SeekBarLabelConfig> {
       this.setThumbnail(this.player.getThumbnail(absoluteSeekTarget));
     }
 
+    if (args.marker) {
+      this.setTitleText(args.marker.marker.title);
+    } else {
+      this.setTitleText(null);
```

Comment:
shouldn't this one be '' as the default for `setTitleText(text = '')` as the type is expecting a string?  
or just this.setTitleText()